daft/dataframe/dataframe.py

```diff
@@ -733,13 +726,9 @@ def repartition(self, num: Optional[int], *partition_by: ColumnInputType) -> "Da
                 "No columns specified for repartition; If you do not require rebalancing of partitions, you may "
```

Comment:
Maybe we should also include in this warning that we will be attempting a `random_shuffle`